Cargo.toml

```diff
@@ -28,7 +28,7 @@ regex = "1"
 serde = { version = "1.0", features = ["derive", "rc"] }
 serde_json = "1.0"
 blake2 = "0.10"
-ssi = "0.9"
+ssi = { version = "0.9", features = ["secp384r1"] }
```

Comment:
Could you please elaborate on why we need this feature for tests?

Reply:
To be able to trigger the unsupported key error.
https://github.com/affinidi/affinidi-did-resolver/pull/33/files#diff-1527caf394ce904addf6f1ed3798fcc5b4c2f65031e4781963b1431b06693856R865